internal/rest/avi_obj_vs.go

```diff
@@ -560,6 +560,8 @@ func (rest *RestOperations) StatusUpdateForVS(restMethod utils.RestMethod, vsCac
 		// }
 		// utils.AviLog.Infof("key: %s Publishing to status queue, options: %v", updateOptions.ServiceMetadata.IngressName, utils.Stringify(statusOption))
 		// status.PublishToStatusQueue(updateOptions.ServiceMetadata.IngressName, statusOption)
+	default:
```

Comment:
This condition will happen in case of insecure SNI when the empty string is returned from the function `serviceMetadataObj.ServiceMetadataMapping("VS")`, right?